### DISCUSSION TITLE
CH01_04. kubectl과 kustomize 설치 (Ubuntu) 3분 53초

### DISCUSSION THREAD

**Discussion by leemk119** [tedilabs/fastcampus-devops#46] [General]
이전 강의는 모두 완료 후 kubectl 설치 과정에서 정상적으로 설치가 되지 않았는지 kubectl 명령어를 확인 할 수가 없습니다.
OS: AWS ubuntu
Source: 강사님 github의 kubectl.sh 실행 

우선 해결은 하였습니다. 강사님의 shell 스크립트의 내용을 하나씩 실행후 정상적으로 명령어 사용이 가능합니다.
하지만 실습중 문제가 될까하여 문의 드립니다.

**Comment by posquit0** [comment]
안녕하세요~ 이슈 올려 주셔서 감사합니다.
`fastcampus-devops/3-docker-kubernetes/env/terraform-aws-ubuntu/ec2-instance/files/install-kubectl.sh` 이 파일인지 경로 확인 가능할까요?

**Comment by leemk119** [comment]
3-docker-kubernetes/env/ubuntu/install-kubectl.sh   이 파일로 기억합니다.  (제가 실수 했을 수도 있습니다.)
(URL:  https://github.com/tedilabs/fastcampus-devops/blob/main/3-docker-kubernetes/env/ubuntu/install-kubectl.sh)
